src/buildHTML.js

```diff
@@ -1203,6 +1198,30 @@ groupTypes.delimsizing = function(group, options) {
             [group.value.mclass]);
 };
 
+groupTypes.verb = function(group, options) {
+    const text = buildCommon.makeVerb(group, options);
+    const body = [];
+    // \verb enters text mode and therefore is sized like \textstyle
+    const newOptions = options.havingStyle(options.style.text());
+    for (let i = 0; i < text.length; i++) {
+        if (text[i] === ' ') {  // Space appear as nonbreaking space
+            // The space character isn't in the Typewriter-Regular font,
+            // so we implement it as a kern of the same size as a character.
+            // 0.524995 is the width of a texttt character in LaTeX.
+            // It automatically gets scaled by the font size.
```

Comment:
You suggested the reverse in #574. 😄 FWIW, TeX implements spaces as skips instead of characters, but I agree it'd be more symmetric (and simpler) to use a space character. I'm not sure where to collect together font issues -- perhaps on new font repo?   Added https://github.com/KaTeX/katex-fonts/issues/1

Reply:
I forgot about that comment.  I think the difference here is that `\verb` is kind of like `\text` and there we use real spaces.  I think what you have here is fine.